From 5c45d61ab6b0919c1c4ab8d520309df24fa3b71d Mon Sep 17 00:00:00 2001
From: John Dias <joaodias@google.com>
Date: Tue, 11 Jul 2017 12:04:08 -0700
Subject: [PATCH 355/420] {chiron,sagit}_defconfig: disable SCHED_AUTOGROUP

Under some workloads, we're seeing periods of freezing
and excessive jank that appear to be partially improved
by disabling the autogroup feature.

Bug: 63331203
Signed-off-by: John Dias <joaodias@google.com>
Change-Id: If61dd69f6a1930158ca7e8c53240042381bc7c77
---
 arch/arm64/configs/chiron_defconfig | 3 ++-
 arch/arm64/configs/sagit_defconfig  | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/configs/chiron_defconfig b/arch/arm64/configs/chiron_defconfig
index 6139cc009af2..8efa97614bb7 100644
--- a/arch/arm64/configs/chiron_defconfig
+++ b/arch/arm64/configs/chiron_defconfig
@@ -16,11 +16,13 @@ CONFIG_RCU_NOCB_CPU_ALL=y
 CONFIG_IKCONFIG=y
 CONFIG_IKCONFIG_PROC=y
 CONFIG_LOG_CPU_MAX_BUF_SHIFT=17
+CONFIG_CGROUPS=y
 CONFIG_CGROUP_DEBUG=y
 CONFIG_CGROUP_FREEZER=y
 CONFIG_CPUSETS=y
 CONFIG_CGROUP_CPUACCT=y
 CONFIG_CGROUP_SCHEDTUNE=y
+CONFIG_CGROUP_SCHED=y
 CONFIG_RT_GROUP_SCHED=y
 CONFIG_SCHED_HMP=y
 CONFIG_SCHED_HMP_CSTATE_AWARE=y
@@ -28,7 +30,6 @@ CONFIG_SCHED_CORE_CTL=y
 CONFIG_NAMESPACES=y
 # CONFIG_UTS_NS is not set
 # CONFIG_PID_NS is not set
-CONFIG_SCHED_AUTOGROUP=y
 CONFIG_SCHED_TUNE=y
 CONFIG_BLK_DEV_INITRD=y
 # CONFIG_RD_XZ is not set
diff --git a/arch/arm64/configs/sagit_defconfig b/arch/arm64/configs/sagit_defconfig
index 6139cc009af2..8efa97614bb7 100644
--- a/arch/arm64/configs/sagit_defconfig
+++ b/arch/arm64/configs/sagit_defconfig
@@ -16,11 +16,13 @@ CONFIG_RCU_NOCB_CPU_ALL=y
 CONFIG_IKCONFIG=y
 CONFIG_IKCONFIG_PROC=y
 CONFIG_LOG_CPU_MAX_BUF_SHIFT=17
+CONFIG_CGROUPS=y
 CONFIG_CGROUP_DEBUG=y
 CONFIG_CGROUP_FREEZER=y
 CONFIG_CPUSETS=y
 CONFIG_CGROUP_CPUACCT=y
 CONFIG_CGROUP_SCHEDTUNE=y
+CONFIG_CGROUP_SCHED=y
 CONFIG_RT_GROUP_SCHED=y
 CONFIG_SCHED_HMP=y
 CONFIG_SCHED_HMP_CSTATE_AWARE=y
@@ -28,7 +30,6 @@ CONFIG_SCHED_CORE_CTL=y
 CONFIG_NAMESPACES=y
 # CONFIG_UTS_NS is not set
 # CONFIG_PID_NS is not set
-CONFIG_SCHED_AUTOGROUP=y
 CONFIG_SCHED_TUNE=y
 CONFIG_BLK_DEV_INITRD=y
 # CONFIG_RD_XZ is not set
-- 
2.15.0

